src/source/raster_dem_tile_worker_source.js

```diff
@@ -16,7 +14,7 @@ class RasterDEMTileWorkerSource {
     loadTile(params: WorkerDEMTileParameters, callback: WorkerDEMTileCallback) {
         const {uid, encoding, rawImageData, padding, buildQuadTree} = params;
         // Main thread will transfer ImageBitmap if offscreen decode with OffscreenCanvas is supported, else it will transfer an already decoded image.
-        const imagePixels = (ImageBitmap && rawImageData instanceof ImageBitmap) ? this.getImageData(rawImageData, padding) : rawImageData;
+        const imagePixels = rawImageData instanceof RGBAImage ? rawImageData : this.getImageData(rawImageData, padding);
```

Comment:
This change breaks terrain in Firefox, which doesn't support `OffscreenCanvas`. This check fails since `rawImageData` is of type `ImageData`.